domain/consensus/datastructures/blockstatusstore/block_status_store.go

```diff
@@ -9,17 +9,19 @@ import (
 	"github.com/kaspanet/kaspad/domain/consensus/utils/lrucache"
 )
 
-var bucket = database.MakeBucket([]byte("block-statuses"))
+var bucketName = []byte("block-statuses")
 
 // blockStatusStore represents a store of BlockStatuses
 type blockStatusStore struct {
-	cache *lrucache.LRUCache
+	cache  *lrucache.LRUCache
+	bucket model.DBBucket
 }
 
 // New instantiates a new BlockStatusStore
-func New(cacheSize int, preallocate bool) model.BlockStatusStore {
+func New(prefix byte, cacheSize int, preallocate bool) model.BlockStatusStore {
```

Comment:
I'd rather if all the `New`s got prefix as byte array already (or if you want to go an extra mile - new type called DBPrefix which is alias to byte array - just make sure MakeBucket is able to get it).

The one who should make sure that the prefix is 1-byte long is the place which creates the prefixes.